### PR TITLE
feature/AT-RSC-UI-UnitTest/CodeCoverage

### PR DESCRIPTION
**Pull request by MajoYanez**
@LuisAT07 @Efcobr-AT @Yairmar

### CODE REVIEW

File: src/test/controllers/admin.controller.test.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/**
+ * AT UNIVERSITY UI - AT Admin Controller Test.
```
Comment (at-internship) [inline src/test/controllers/admin.controller.test.js]: AT RESOURCES UI

File: src/test/controllers/admin.controller.test.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ * @author @at-internship
+ * @version 1.0
+ */
+
+// Constants
+const sinon = require("sinon");
+const expect = require("chai").expect;
+
+// AT Admin Controller
+const adminC = require("../../controllers/admin.controller");
```
Comment (at-internship) [inline src/test/controllers/admin.controller.test.js]: const adminController

File: src/test/services/at-resources.service.test.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
+/**
+ * AT UI MOCKS - AT University Service API Test.
```
Comment (at-internship) [inline src/test/services/at-resources.service.test.js]: AT RESOURCES UI

File: src/test/services/at-resources.service.test.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+ * Copyright 2020 AgileThought, Inc.
+ * 
+ * Integration Test for at-university-api endpoint.
```
Comment (at-internship) [inline src/test/services/at-resources.service.test.js]: Integration Test for at-resources-api endpoint.

File: src/test/services/at-resources.service.test.js [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+ * @version 1.0
+ */
+
+// Constants
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const expect = require('chai').expect;
+chai.use(chaiHttp);
+
+// AT University Service API
```
Comment (at-internship) [inline src/test/services/at-resources.service.test.js]: AT Resources Service API

**Review by at-internship — CHANGES_REQUESTED** [review]

File: src/test/services/at-resources.service.test.js (hunk 4 above)
Comment (at-internship) [reply src/test/services/at-resources.service.test.js]: AT RESOURCES UI - AT Resources Service API Test.

**Review by Yairmar — APPROVED** [review]

File: package.json [hunk 6]
```diff
@@ -21,14 +23,18 @@
     "dependencies": {
         "axios": "^0.20.0",
         "bcryptjs": "^2.4.3",
+        "chai": "^4.2.0",
         "connect-flash": "^0.1.1",
```
Comment (at-internship) [inline package.json:28]: chai-http dependency is missing "chai-http": "^4.3.0",

**Review by LuisAT07 — APPROVED** [review]

**Review by at-internship — APPROVED** [review]
Approved

**Pull request merged**